service/app/poller.go

```diff
@@ -350,9 +351,22 @@ func processSendableTransaction(ctx context.Context, app *App, logger *log.Entry
 	if err = app.service.flowClient.SendTransaction(ctx, *tx); err != nil {
 		err = fmt.Errorf("error while sending transaction: %w", err)
 
-		t.State = common.TransactionStateFailed
 		t.Error = err.Error()
 
+		if strings.Contains(t.Error, common.TransactionRPCErrorString) {
```

Comment:
Should all rpc errors be retried or only the deadline exceeded error is worth retrying?

Reply:
Ah, good catch - let me look into this

Reply:
Hey @EricLin-wk I think we should be good for now handling all rpc errors 👀 
These rpc errors are also `context deadline exceeded errors`.